### ISSUE TITLE
Display inbound rejections to the user.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [anonaddy/anonaddy#672]
As explained at https://github.com/anonaddy/anonaddy/discussions/670#discussion-7025806, I've had a message fail to reach a recipient of mine, which was luckily caught by https://app.addy.io/failed-deliveries (despite me having `addy.io` set as an excluded domain). However, this has made me realise that having a middleman just pushes the problem back:

> Yes that banner is added when a message looks like spam or has a DMARC policy of none/quarantine that has failed. But if the domain had a DMARC policy of reject and it fails then it will be rejected.

Unfortunately, the functionality to verify that AnonAddy isn't blocking anything I care about doesn't exist, but is possible to provide (as https://github.com/anonaddy/anonaddy/discussions/670#discussioncomment-10303366 explains):

> I may be able to parse the mail logs in order to also show inbound rejections in the future.

Consequently, I'd like to be able to see inbound rejections (alongside the aforementioned existent outbound rejection list) to ensure that Addy isn't blocking mail *from senders that I care about*.

**Comment by hlbatt** [comment]
Yes! addy.io has started blocking emails from meta to my aliases. It would be useful to be able to whitelist them myself rather than have to contact someone to fix it.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/anonaddy/anonaddy/issues/672#issuecomment-2494031320

@hlbatt, I've missed e-mails from the Ministry of Defence Police due to this, too... It's a fairly significant flaw of most mail providers. [^1]

[^1]: https://www.reddit.com/user/rokejulianlockhart/comments/1ehld3w/comment/lgiodth/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Comment by willbrowningme** [comment]
This has now been added and can be seen on the [failed deliveries](https://app.addy.io/failed-deliveries) page.

**Comment by RokeJulianLockhart** [comment]
@willbrowningme, thank you! I've now almost no reservations about recommending Addy to others. However, one concern remains – that I can't see the rejected message. This matters when the sender is absent, as the undermentioned example demonstrates:

<p></p><img width="1949" height="346" alt="Image" src="https://github.com/user-attachments/assets/f8cf1b6f-39dd-4a8d-a038-2c1e87fb1c60" /><p></p>

Would having a “Download” option available for these be worth another issue?

**Comment by willbrowningme** [comment]
@RokeJulianLockhart Having a download option for these inbound rejections is not possible as the messages are never even accepted by the addy.io mail servers.

That example in your screenshot showing `<>` as the sender means that the envelope from address was empty and it was certainly spam.
